brew-build.sh

```diff
@@ -29,19 +29,23 @@ BREW_FROM_SRC_PACKAGES=(
   automake
   bzip2
   flex
-  gcc
   icu4c
   libtool
-  libuuid
   ninja
-  openssl
   readline
-  s3cmd
```

Comment:
It would be better to install it outside of Homebrew/Linuxbrew that we use for building C/C++ code, because it has a lot of dependencies that we don't need otherwise.